src/lumen/activity/field/number.hpp

```diff
@@ -12,7 +12,7 @@ class Number : public Field<uint> {
      * \param increment Interval by which the value of the field change each
      * time `increase()` or `decrease()` is called.
      */
-    explicit Number(uint increment = 1);
+    explicit Number(uint start_value = 0, uint increment = 1);
```

Comment:
- [ ] Add a `start_value_` private data member and initialize it to `start_value`.

This will be helpful when storing the activity